issue_metrics.py

```diff
@@ -159,6 +161,9 @@ def get_per_issue_metrics(
                         )
             elif issue.state == "open":  # type: ignore
                 num_issues_open += 1
+            if env_vars.hide_created_at is False:
+                issue_with_metrics.created_at = issue.created_at
+
```

Comment:
```suggestion
if not env_vars.hide_created_at:
   issue_with_metrics.created_at = issue.created_at
```

we don't need the above in both the if and else.  it can go once here. wdyt?